firebase.json

```diff
@@ -321,6 +321,7 @@
       { "source": "/go/detect-memory-leaks", "destination": "https://docs.google.com/document/d/1hvRVpULPnwGjBc5nw52iSaDfB8hjIzbaQuECHmKvDrY/edit", "type": 301 },
       { "source": "/go/developing-plugins", "destination": "/packages-and-plugins/developing-packages#plugin", "type": 301 },
       { "source": "/go/devintel-q3-2023", "destination": "https://docs.google.com/document/d/1VGux5IxePO8lN_7kUuzYJtcJ76fkldr_TJv4R0tciYk", "type": 301 },
+      { "source": "/go/devintel-q1-2024", "destination": "https://docs.google.com/document/d/15W9wsqzFcidXR12DiPH5r5ZDcZw_wDa7Zi9CAAe34Y4", "type": 301 },
```

Comment:
The link doesn't seem to be viewable with just the link.

Reply:
I'm curious why the link directly above this one works without any additional args? Embedded images? 
 
 Either way, thanks for catching.

Reply:
I haven't figured out definitively when a resource key is needed or not. I thought it was related to the doc owner's organization configuration, but it seems inconsistent :P